packages/@adobe/spectrum-css-temp/components/searchwithin/index.css

```diff
@@ -64,3 +64,12 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+/* this is specific to handling default width */
+.spectrum-SearchWithin-container {
+  /* ensure we have higher specificity than .spectrum-Field */
+  &.spectrum-SearchWithin-container {
+     /* override the default width of the field container, only with labelPosition=top. */
+     --spectrum-field-default-width: var(--spectrum-global-dimension-size-3000);
```

Comment:
why does this apply only with labelPosition="top"? I noticed that changing the picker selection for `SearchWithin labelPosition=side` still causes the entire SearchWithin to change size rather than just the picker itself

Reply:
Perhaps we can have this class setup a default width explicitly that gets overwritten if the user provides a customWidth?

Reply:
I think we may have to solve that one later https://www.chromatic.com/build?appId=5f0dd5ad2b5fc10022a2e320&number=100